src/main/resources/cms.conf

```diff
@@ -121,6 +117,12 @@ cms.jobs.configuredJobs=[
         "repeatCount":  -1,
         "repeatInterval": 10,
         "repeatTimeUnit": "minutes"
+    },
+    {
+        "jobClassName": "HystrixMetricsProcessingJob",
+        "repeatCount":  -1,
+        "repeatInterval": 1,
```

Comment:
@tlisonbee what should the default interval be 10 seconds?

Reply:
I'm not sure, 10 at minimum

Reply:
- [ ]  Start with 10 seconds